src/pages/MetricDetail.svelte

```diff
@@ -54,7 +54,11 @@
     additionalInfo
   ) {
     if (metricType === "event") {
-      return getGleanEventQuery(table, additionalInfo);
+      // Although events might come from other pings, we override that and
+      // generate SQL just for the `events_stream`. This changes
+      // `some_database.table` to `some_database.events_stream`.
```

Comment:
I'm not really sure if we want to retain the old SQL code generation. Does it make any sense? This is quite a hack @badboy

Reply:
For events that are not in an events ping this will be incorrect with this change, because they don't end up in the stream just yet.

But otherwise it's fine; no point in keeping the legacy way around

Reply:
Ok, what about this? We retain the old query for non 'events' pings until this gets fixed.